neuralprophet/forecaster.py

```diff
@@ -805,16 +805,17 @@ def split_df(self, df, freq, valid_p=0.2, local_modeling=False):
         Args:
             df (pd.DataFrame): data
             freq (str):Data step sizes. Frequency of data recording,
-                Any valid frequency for pd.date_range, such as '5min', 'D' or 'MS'
+                Any valid frequency for pd.date_range, such as '5min', 'D', 'MS' or 'auto' (default) to automatically set frequency.
             valid_p (float): fraction of data to use for holdout validation set
                 Targets will still never be shared.
 
         Returns:
             df_train (pd.DataFrame):  training data
             df_val (pd.DataFrame): validation data
         """
-        df = df.copy(deep=True)
+        df = df.copy()
```

Comment:
@ourownstory This should be solved by the df copy PR. I had to change it here too so the Split df feature could work.